deployment/streaming/docker-compose.deploy.yml.tpl

```diff
@@ -0,0 +1,44 @@
+version: '3.0'
+services:
+  augdiff-stream:
+    image: ${ECR_REPO}:latest
+    command: >
+      /spark/bin/spark-submit --driver-memory 4096m --class osmesa.analytics.oneoffs.AugmentedDiffStreamProcessor /opt/osmesa-analytics.jar
```

Comment:
Does it make sense to submit these to a shared local master + worker (w/ reservations on partial capacity) rather than running 2 in "local" mode? (For logging / better capacity management??)